api-reference/10 UI Components/dxNumberBox/1 Configuration/format.md

```diff
@@ -8,13 +8,114 @@ default: ''
+
+- Custom format string    
+
+    Use Unicode Locale Data Markup Language (LDML) patterns to specify a custom format string. An LDML pattern consists of wildcard characters and characters displayed as is. The **format** property supports the following wildcard characters:
+
+    <table class="dx-table">
+        <tr>
+        <th>Format character</th>
+        <th>Description</th>
+        </tr>
+        <tr>
+        <td>0</td>
+        <td>A digit. Displays '0' if the formatted number doesn't have a digit in that position.</td>
+        </tr>
+        <tr>
+        <td>&#35;</td>
+        <td>
+                Any number of leading digits, a single digit, or nothing. If this character goes first in the format string, it can match multiple leading digits (before the decimal point). Consequent characters match a single digit. If the formatted number doesn't have a digit in the corresponding position, displays nothing.</br> 
+                For example, if you apply format "#0.#" to "123.45", the result is "123.4".
+        </td>
+        </tr>
+        <tr>
+        <td>.</td>
+        <td>
+                A decimal separator. </br>
+                Actual character depends on locale.
+        </td>
+        </tr>
+        <tr>
+        <td>,</td>
+        <td>
+                A group separator. </br>
+                Actual character depends on locale.
+        </td>
+        </tr>
+        <tr>
+        <td>%</td>
+        <td>
+                The percent sign. Multiplies the input value by 100.
+        </td>
+        </tr>
+        <tr>
+        <td>;</td>
+        <td>Separates positive and negative format patterns. This character is optional. If you do not use it, then you specify a common format for positive and negative numbers. Negative numbers simply display a minus ("-") prefix. </td>
+        </tr>
+        <tr>
+        <td>Escape characters</td>
+        <td>
+                You can display the special characters above as literals if you enclose them in single quotation marks. <br/>
+                For example, '%'.
+        </td>
+        </tr>
+        <td>Other characters</td>
+        <td>
+                You can add any literal characters to the beginning or end of the format string.
+        </td>
+        </tr>
+    </table>
+
+        <!-- tab: JavaScript -->
+        const largeNumber = 123456.789;
+
+        // Limit the decimal part to one digit
+        format: "#0.#" // 123456.7
+
+        // Add a group separator
+        format: ",##0.###" // 123,456.789
+        
+    The examples below show different ways to apply percentage formatting to decimals. Use caution if your format strings starts with a zero ('0'), because the formatted number may lose leading digits.
```

Comment:
```suggestion
    The examples below show different ways to apply percentage formatting to decimals. Use caution if your format string starts with a zero ('0'), because the formatted number may lose leading digits.
```